components/pages/CVGuide.tsx

```
@@ -0,0 +1,768 @@
+            <div>
+              <p className="italic text-gray-700 dark:text-gray-300"><br />Reviewed by apprentices and hiring managers at Accenture, HSBC, and Digital Catapult.</p>
+            </div>
+          </motion.div>
+
+          <Section
+            id="why"
+            icon={<FileQuestion />}
+            title="Why Your CV Sucks"
+            content={
+              <div className="text-gray-700 dark:text-gray-300">
+                <p>In the world of apprenticeship applications, first impressions are everything. Your Curriculum Vitae (CV) is your first opportunity, your first chance to leave an imprint of yourself, and traverse through the application process. A well-crafted CV is not just a list of your experiences—it's a powerful story of who you are, what you've achieved, and where you're going. </p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)